packages/uniforms/src/filterDOMProps.js

```diff
@@ -2,6 +2,7 @@ const unwantedProps = [
     // These props are provided by BaseField
     'changed',
     'changedMap',
+    'checkbox',
```

Comment:
These are global, so it's a breaking change. I'd rather go with destructuring props in `BoolField` - it won't be needed then.